lib/module.js

```diff
@@ -460,29 +461,23 @@ Module.runMain = function() {
 };
 
 Module._initPaths = function() {
-  const isWindows = process.platform === 'win32';
+  modulePaths = [];
```

Comment:
Looking at https://github.com/zertosh/node/blob/module-improv/lib/module.js#L41 I don't think it is undefined. 
If `_initPaths` is only called once there is no need to assign `modulePaths` again.